### PR TITLE
Update Tempo to latest 2.4.0

### PR DESCRIPTION
**Pull request by george-angel**

### CODE REVIEW

File: tempo/base/kustomization.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ resources:
 images:
   - name: tempo
     newName: grafana/tempo
-    newTag: 2.4.0-rc.0
+    newTag: 2.4.0
```
Comment (matthewhughes-uw) [inline tempo/base/kustomization.yaml:29]: I'm assuming there aren't many significant changes since the release candidate?

**Review by matthewhughes-uw — APPROVED** [review]

File: tempo/base/kustomization.yaml (hunk 1 above)
Comment (george-angel) [reply tempo/base/kustomization.yaml:29]: No changes: https://github.com/grafana/tempo/compare/v2.4.0...v2.4.0-rc.0

Comment (matthewhughes-uw) [reply tempo/base/kustomization.yaml:29]: > No changes: [grafana/tempo@v2.4.0...v2.4.0-rc.0](https://github.com/grafana/tempo/compare/v2.4.0...v2.4.0-rc.0)

I think your range is backwards: https://github.com/grafana/tempo/compare/v2.4.0-rc.0...v2.4.0

Comment (george-angel) [reply tempo/base/kustomization.yaml:29]: Oh tyty!

Doesn't look like it, I also checked the notes separately.

**Pull request merged**